book/src/integrate/web.md

```diff
@@ -1,4 +1,41 @@
 # Integrating with Web
 
-As of writing, Web support is underway and tracked by [fzyzcjy/flutter_rust_bridge#315](https://github.com/fzyzcjy/flutter_rust_bridge/issues/315).
-Consider leaving comments and reactions to let us know of the demand for this feature!
+Refer to the [Web setup](../template/setup_web.md) page for required installables.
+
+Once you have installed the required dependencies, you will need to create a wrapper
+to consume the bridge files. In the case of `DynamicLibrary`
+you only needed to supply the path to the binary,
+but to import a WASM module you need to:
+
+- Create a script tag to the JS file generated by `wasm_bindgen` and insert it into the document;
+- Invoke the `wasmModule` initializer defined in the
+  web bridge;
+- And finally, create the implementation class.
+
+Create a Dart file and copy these lines to it:
+
+```dart
+import 'bridge_generated.web.dart';
+export 'bridge_definitions.dart';
+
+import 'dart:html';
+
+// Path to the wasm_bindgen generated files
+const root = 'pkg/native';
+final api = NativeImpl.wasm(_initWasm());
+
+Future<WasmModule> _initWasm() {
```

Comment:
Is it possible that we provide this function in our `frb_dart` package, such that users can directly use it. And at the same time, if they need customization, they can copy-and-paste and modify (or just make a pr to the original function)

Reply:
Other implementations of the Rust WASM worker pool (e.g. wasm_thread) actually allows importing ES modules instead of bare script files. This is one area I don't feel comfortable sugar-coating over as it depends greatly on the user's setup, and as a library we have absolutely no control over what gets emitted.

Reply:
That said, if we put a disclaimer I think I can allow it for the sake of convenience.

Reply:
I see. What about `initWasmAAA` where AAA is the name of wasm impl for this case? In any case a util function can save some time of the users